<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tch_manage</class>
 <widget class="QMainWindow" name="tch_manage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AtQlas Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>562</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border-image:url(:/atqres_ico/tchm.png)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="lesson_table">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>230</y>
       <width>581</width>
       <height>192</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget 
{background-color: transparent;
}
QScrollBar:vertical {
    background: url(images/pure_red.png);
    width: 0px;
    margin: 0px 0 0px 0;
}
QScrollBar::handle:vertical {
    background: rgb(195, 195, 195);
    min-height: 30px;
    margin: 0 1px 0 2px;
    border-radius: 3px;
    border: none;
}

QScrollBar:horizontal {
    height: 0px;
    margin: 0px 0 0px 0;
}
QScrollBar::handle:horizontal {
    background: rgb(195, 195, 195);
    min-width: 20px;
    margin: 2px 0px 1px 0px;
    border-radius: 3px;
    border: none;
}
QHeaderView::section
{
border-image:url(:/atqres_ico/pure_red.png);
height:30px;
spacing: 20px;
background-color:rgb(39,49,58);
color: white;

text-align: right;
font-family: PT Mono;
font-size:12px;
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_button">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>250</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border-image:url(:/atqres_ico/minuslite.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="add_new_button">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>330</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border-image:url(:/atqres_ico/pluslite.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="add_all_button">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>450</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border-image:url(:/atqres_ico/plusliteall.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="add_selected_button">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>450</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border-image:url(:/atqres_ico/pluslitelite.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="filter_edit">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>205</y>
       <width>217</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border-image:url(:/atqres_ico/pure_gray_7.png)</string>
     </property>
     <property name="placeholderText">
      <string>Type to filter...</string>
     </property>
    </widget>
    <widget class="QLabel" name="tid_label">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>309</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TID</string>
     </property>
    </widget>
    <widget class="QLabel" name="school_label">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>130</y>
       <width>309</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>school</string>
     </property>
    </widget>
    <widget class="QLabel" name="name_label">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>80</y>
       <width>309</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>name</string>
     </property>
    </widget>
    <widget class="QLabel" name="tid_label_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>309</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="school_label_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>309</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>School</string>
     </property>
    </widget>
    <widget class="QLabel" name="name_label_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>309</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="back">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border-image:url(:/atqres_ico/cock_back.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>230</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border-image:url(:/atqres_ico/lesson_db_all.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="refresh">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>310</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border-image:url(:/atqres_ico/refresh.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
